src/lib.rs

```diff
@@ -247,7 +248,7 @@ impl<'a> RobotFileParser<'a> {
         let client = Client::new();
         let request = client.get(self.url.clone())
             .header(UserAgent(USER_AGENT.to_owned()));
-        let mut res = match request.send() {
+        let res = match request.send() {
```

Comment:
also I have a question why mut is no longer required ? if ```from_response``` accepts ```mut``` only ? (since required by read_to_string) 

```rust
pub fn from_response(&self, mut response: Response) {
  let mut buf = String::new();
  response.read_to_string(&mut buf).unwrap();
```